css/header.scss

```diff
@@ -47,7 +77,7 @@
   justify-content: center;
   align-items: center;
   flex-direction: column;
-  background-color: rgb(39,146,93);
+  background-color: rgb(39, 146, 93);
```

Comment:
Try to avoid mixing different color systems in one file without need. Use HEX or RGB, but not both. The exception is when you're using HEX and RGBA for colors with Alpha channel.

Reply:
Completely agree, just being lazy and then forgot to change it :D